build.psm1

```diff
@@ -93,14 +93,10 @@ function Start-DocumentationBuild
     $docsPath = Join-Path $projectRoot docs
     $markdownDocsPath = Join-Path $docsPath markdown
     $outputDocsPath = Join-Path $destinationDir en-US
-    $requiredVersionOfplatyPS = 0.9
-    #$modInfo = new-object Microsoft.PowerShell.Commands.ModuleSpecification -ArgumentList @{ ModuleName = "platyps"; ModuleVersion = $requiredVersionOfplatyPS}
-    #if ( $null -eq (Get-Module -ListAvailable -FullyQualifiedName $modInfo))
-    if ( $null -eq (Get-Module -ListAvailable platyPS))
+    if ( $null -eq (Get-Module -ListAvailable platyPS) -or ((Get-Module -ListAvailable platyPS | Select-Object -First 1).Version -lt [version]0.12))
```

Comment:
Style: Got a space on the left of the `$null` but no space on the right of the `0.12`.

`gmo -List` is being called twice here and only looks at the first module (`gmo -List` returns module in essentially random order, so that might not be the check we want).

Maybe something like:

```powershell
$platyPS = Get-Module -ListAvailable platyPS |
    Where-Object Version -ge '0.12' |
    Sort-Object Version |
    Select-Object -First 1

if (-not $platyPS)
{
    Write-Verbose -verbose "platyPS module not found or below required version of 0.12, installing the latest version."
    Install-Module -Force -Name platyPS -Scope CurrentUser
}
```

Reply:
Refactored that slightly to use a FullyQualifiedName

Reply:
Sorry, but `Get-Module -ListAvailable @{ ModuleName = 'platyPS'; ModuleVersion = '0.12.0' }` does not work on my machine using either WPS or PSCore, I will refactor it to call `gmo` only once though and sort by version